stable/ambassador/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 This file documents all notable changes to Ambassador Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v5.0.0
+
+### Breaking changes
+
+- Helm 3 support for CRDs was added. Specifically, the CRD templates were moved to non-templated files in the `/crds` directory, and to keep Helm 2 support they are globbed from there by `/templates/crds.yaml`. However, because Helm 3 CRDs are not templated, the labels have necessarily changed. Because of this, you must manually delete any ambassador CRDs installed by previous versions of this chart before upgrading
```

Comment:
```suggestion
- Helm 3 support for CRDs was added. Specifically, the CRD templates were moved to non-templated files in the `/crds` directory, and to keep Helm 2 support they are globbed from there by `/templates/crds.yaml`. However, because Helm 3 CRDs are not templated, the labels have necessarily changed. Because of this, you must manually delete any ambassador CRDs installed by previous versions of this chart before upgrading

**Note** If upgrading a helm 2 based install no action is needed, previously installed CRDs will not be modified.
```

Reply:
You are right. I just tested - upgrading to `5.0.0` with Helm 2 does not modify the CRDs, so I'm changing the wording. Mostly what you suggested, but modified my initial line to be more accurate in 4da3a85